go/fn/Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: fix vet fmt test lint
+
+GOPATH := $(shell go env GOPATH)
+OUT_DIR := .out
+
+all: fix fmt test lint
+
+fix:
+	go fix ./...
+
+fmt:
+	go fmt ./...
+
+lint:
+	(which $(GOPATH)/bin/golangci-lint || go get github.com/golangci/golangci-lint/cmd/golangci-lint)
```

Comment:
Maybe pin to a version to make it more stable.

Reply:
sg.